### ISSUE TITLE
Cannot succeed to install on windows10

### ISSUE THREAD

**Issue by yzf123456** [r9y9/pylibfreenect2#60]
![image](https://user-images.githubusercontent.com/43875437/56334478-04e6ca00-61cb-11e9-9308-36b9fc8c460c.png)
I don't understand this? i have installed libfreenect2 correctly and confirm that Protonect works.

**Comment by r9y9** [comment]
What’s unclear with you?

**Comment by yzf123456** [comment]
i don't know how to set the environmental,because i cannot find /usr/local/ on my computer,
and i want to know if LIBFREENECT2_INSTALL_PREFIX=C:\Users\DELL\Desktop\pylibfreenect2-master is correct? Thank you .

**Comment by r9y9** [comment]
That's incorrect. Please do make sure that `${LIBFREENECT2_INSTALL_PREFIX}include/libfreenect2/config.h` exists. Replace `/` to `\` if you use windows.

**Comment by yzf123456** [comment]
OK, i find the config.h in C:\Users\DELL\Desktop\libfreenect2-master\build\install\include\libfreenect2,so
i should set the environmental variable LIBFREENECT2_INSTALL_PREFIX=C:\Users\DELL\Desktop\libfreenect2-master\build\install?thank you,i am freshman.

**Comment by NIravMeghani** [comment]
![image](https://user-images.githubusercontent.com/56798346/79036829-a986c680-7be9-11ea-9945-a841cb7fbb8a.png)
![image](https://user-images.githubusercontent.com/56798346/79036851-d935ce80-7be9-11ea-8e99-da114cdb5d55.png)
any solution...?

**Comment by Selar007** [comment]
A small guide how to install on windows 10:
https://stackoverflow.com/questions/67902941/how-to-call-multiple-kinect2-device-with-pykinect2/67914523#67914523
